### PR TITLE
Build registry for detokeniser

### PR DESCRIPTION
**Pull request by gabechu**
### Description
Add registry for detokeniser. This is the same as #8.

### Context
Detokeniser is used to untokenise a sentence. Datasets like `CONLL` are organised and segmented by tokens, where every sentence is broken into tokens accompanying with POS, entity label ect. However, many ML models in NER would not accept tokens as an input, instead requiring a string as an input. Detokeniser handles the conversion from tokens to a string, so that evaluations can be done bridging different kinds of inputs.

#### Checklist
- [x] Added **tests** for changed code.
- [x] Updated **documentation** for changed code.

### CODE REVIEW

**Comment by kerrychu-zen** [comment]
`detokenisers` test?

File: pii_recognition/registers/detokeniser_registry_test.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from .detokeniser_registry import DetokeniserRegistry
+
+
+def test_DetokeniserRegistry():
+    actual = DetokeniserRegistry()
+    assert isinstance(actual, dict)
+    assert len(actual.keys()) > 0
+    assert len(actual.keys()) == len(actual.values())
```
Comment (ePak) [inline pii_recognition/registers/detokeniser_registry_test.py:8]: When this would actually fail?

**Comment by gabechu** [comment]
> `detokenisers` test?

It's hidden deep in the middle of the many files and Eric found it!

File: pii_recognition/registers/detokeniser_registry_test.py (hunk 1 above)
Comment (gabechu) [reply pii_recognition/registers/detokeniser_registry_test.py:8]: Good point. Will remove it.

**Comment by gabechu** [comment]
💇 @ePak @kerrychu-zen

**Comment by gabechu** [comment]
This PR is closed since we refactored the registry. Now, it would initialised in a different way which required base classes to be passed as parameters.

**Pull request closed without merging**